contribute/contribute-pull-requests/add_hook.md

```diff
@@ -0,0 +1,18 @@
+# How to add a new Hook
+Adding a new hook is a process that can be done in only three steps.
+
+## 1) Add a call to Hook::exec()
+
+As you can see on this [PR](https://github.com/PrestaShop/PrestaShop/pull/34431/files), adding a simple Hook::exec() statement is enough to add a new hook. In order for it to be complete, there is a couple more steps to do.
```

Comment:
```suggestion
As visible in [this PR](https://github.com/PrestaShop/PrestaShop/pull/34431/files), the first step is to add a `Hook::exec()` in the code where you want to execute it. This will execute the code from the modules hooked into your `Hook`.
```